uninstall.php

```diff
@@ -1,17 +1,18 @@
 <?php
+
```

Comment:
Do you store any kind of sensitive info with this plugin in the database? If so, you should clean this up on the uninstall (and add the export and delete functionality using the built-in WP functions)

Reply:
Not really, all the logic for that should be implemented on per-project basis. There's literally not a single option / postmeta we save in forms alone.